### ISSUE TITLE
Please add a sound too

### ISSUE THREAD

**Issue by kennethaar** [sawhney17/logseq-todo-exploder#3]
Love your dopamine plugin. Could you please add a sound too?

**Comment by Julian-1-2-3-4-5** [comment]
Yeah i would really appreciate that as well: maybe a higher tone "pling" like metal "clinging" on metal?
